clients/ptranslator/src/main/java/org/hawkular/metrics/clients/ptrans/PTrans.java

```diff
@@ -60,7 +60,7 @@
  * @author Thomas Segismont
  */
 public class PTrans {
-    private static final Logger LOG = LoggerFactory.getLogger(PTrans.class);
+    private static final PTransLogger log = PTransLogging.getPTransLogger(PTrans.class);
```

Comment:
Not sure what you mean.

The annotation processor just creates an implementation of the 
PTransLogger interface.

If you want the PTrans class to have access to PTransLogger methods, you 
have no choice but to declare the typed logger, not a generic JBoss 
Logging logger.

Does that answer your question?